h2o-py/h2o/backend/connection.py

```diff
@@ -638,7 +638,9 @@ def _log_start_transaction(self, endpoint, data, json, files, params):
         msg += "[%s] %s\n" % (time.strftime("%H:%M:%S"), endpoint)
         if params is not None: msg += "     params: {%s}\n" % ", ".join("%s:%s" % item for item in viewitems(params))
         if data is not None:   msg += "     body: {%s}\n" % ", ".join("%s:%s" % item for item in viewitems(data))
-        if json is not None:   msg += "     json: %s\n" % json.dumps(json)
+        if json is not None:
+            import json as j
```

Comment:
this was a piece of code that existed but never used before
the bug was that `json` is a variable name, and we also tried to use that as a module name. Instead the module ought to be imported by another name in order to prevent name clash (previously we forgot to import the module at all).